File: Sis104Avance/app/src/main/java/com/example/sis104avance/MainActivity.java
package com.example.sis104avance;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

import com.example.sis104avance.ejercicios.EjerciciosActivity;
import com.example.sis104avance.examenes.ExamenActivity;
import com.example.sis104avance.graficos.GraficosActivity;

public class MainActivity extends AppCompatActivity {

    Button button_Ejercicio, button_Examen, button_BaseDatos, button_Grafico, button_Salir;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main), (v, insets) -> {
            Insets systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars());
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom);
            return insets;
        });

        button_Ejercicio = findViewById(R.id.button_Ejercicios);
        button_Examen = findViewById(R.id.button_Examenes);
        button_BaseDatos = findViewById(R.id.button_BaseDatos);
        button_Grafico = findViewById(R.id.button_Graficos);
        button_Salir = findViewById(R.id.button_Salir);

        button_Ejercicio.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, EjerciciosActivity.class);
                startActivity(intent);
            }
        });

        button_Examen.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, ExamenActivity.class);
                startActivity(intent);
            }
        });

        button_Grafico.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, GraficosActivity.class);
                startActivity(intent);
            }
        });

        button_Salir.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                finishAffinity();
            }
        });
    }
}

File: trabajoSIS104/app/src/main/java/com/example/trabajosis104/MainActivity.kt
package com.example.trabajosis104

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import com.example.trabajosis104.baseDatos.BaseDatosActivity
import com.example.trabajosis104.ejercicios.EjerciciosActivity
import com.example.trabajosis104.ejercicios.Grafico2dActivity
import com.example.trabajosis104.examen.ExamenActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }

        val buttonEjercicios = findViewById<Button>(R.id.buttonEjercicios)
        val buttonExamen = findViewById<Button>(R.id.buttonExamen)
        val buttonBaseDatos = findViewById<Button>(R.id.buttonBaseDatos)
        val buttonGrafico = findViewById<Button>(R.id.buttonGrafico)

        buttonEjercicios.setOnClickListener {
            val intent = Intent(this, EjerciciosActivity::class.java)
            startActivity(intent)
        }
        buttonExamen.setOnClickListener {
            val intent = Intent(this, ExamenActivity::class.java)
            startActivity(intent)
        }


        buttonBaseDatos.setOnClickListener {
            val intent = Intent(this, BaseDatosActivity::class.java)
            startActivity(intent)
        }

        buttonGrafico.setOnClickListener {
            val intent = Intent(this, Grafico2dActivity::class.java)
            startActivity(intent)
        }

    }
}